Formula/kn.rb

```diff
@@ -0,0 +1,37 @@
+class Kn < Formula
+  desc "Command-line interface for managing Knative Serving and Eventing resources"
+  homepage "https://github.com/knative/client"
+  url "https://github.com/knative/client.git",
+      tag:      "v0.24.0",
+      revision: "0bbb3ec9d92665d372d8393ef3036ba7e0c069a4"
+  license "Apache-2.0"
+
+  depends_on "bash" => :build
+  depends_on "go" => :build
+
+  def install
+    ENV["CGO_ENABLED"] = "0"
+    ENV["GO111MODULE"] = "on"
+    ENV["TAG"]         = "v#{version}"
+
+    system "./hack/build.sh", "--fast"
+
+    bin.install "kn"
+  end
+
+  test do
+    list_plugins_output = shell_output("#{bin}/kn plugins list")
+    assert_match("No plugins found.", list_plugins_output)
```

Comment:
This is dangerous especially because you can't guarantee that a user has a plugin installed.

For the test, I just would use `kn version` and assert on the version number and possibly on the git commit hash (that you have added above)

Reply:
makes sense.. will remove